src/modules/userInterface.js

```diff
@@ -0,0 +1,26 @@
+import MealAPI from './mealAPI.js';
+
+const recipeList = document.querySelector('.recipes__list');
+export default class UserInterface {
```

Comment:
**OPTIONAL**
Implementing a class that only utilizes static methods may be good for class templates or pure methods to be available. But in this case, we may only need a manipulable object. Please consider using a subclass for creating the user interface so retrieved items may be stored in local variables and avoid any possible excess calls to the API.